diagnostics.yml

```diff
@@ -415,6 +432,15 @@ stages:
                 targetPath: '$(Build.SourcesDirectory)/artifacts/bin/OSX.x64.Release'
               condition: succeeded()
 
+            # MacOS arm64 download
```

Comment:
This only runs when not in PR. Hence this won't work with how you currently set up the build (only build on PR). Also the managed bits only build as netcoreapp2.1. The SDK will choke on that now being a known RID to the framework. We don't have a host for the tools that's OSX-arm64, no shims, and the tools will ultimately fail to start.

Reply:
The initial prompt for this comment was reverted.
I expect the tools managed bits are built for `any` cpu.  

I wouldn't be surprised if the missing SDK is a bit of an issue, but it is the same issue we face with linux-musl-arm64 which doesn't have an SDK.  We have to install on a different platform and copy to the real target....  

I am not convinced I fully understood your comment....

Reply:
The problem is not the managed DLLs. The problem is global tools require a host - I believe it's an apphost shim. Isn't this one native and we'd need one for OSX-arm64? This one wouldn't be in the SDK nor would it be downloadable for the target RID and TargetFramework.

Reply:
The apphost is part of the installer in the dotnet/runtime.  I am not sure if that is compling yet.  I think it is....